test/e2e/cmd/run/eventlog.go

```diff
@@ -106,7 +106,7 @@ func (el *eventLogger) runEventProcessor() {
 			return
 		}
 
-		evtObj, exists, err := el.eventInformer.GetIndexer().GetByKey(key.(string)) //nolint:forcetypeassert
+		evtObj, exists, err := el.eventInformer.GetIndexer().GetByKey(key) //nolint:forcetypeassert
```

Comment:
I think this can be removed since there isn't a type assertion anymore

```suggestion
		evtObj, exists, err := el.eventInformer.GetIndexer().GetByKey(key)
```